.azuredevops/modulePipelines/ms.appconfiguration.configurationstores.yml

```diff
@@ -28,6 +28,7 @@ trigger:
   paths:
     include:
       - '/.azuredevops/modulePipelines/ms.appconfiguration.configurationstores.yml'
+      - '/.azuredevops/modulePipelines/ms.network.privateendpoints.yml'
```

Comment:
General: Shouldn't we trigger on cross referenced module changes (e.g.,
`'/modules/Microsoft.Network/privateEndpoints/*'`) instead of triggering on cross referenced modules pipelines (e.g., `'/.azuredevops/modulePipelines/ms.network.privateendpoints.yml'` ) ?

Reply:
Yes 😄

Reply:
Updated & tested